R/utils_insert_db.R

```diff
@@ -150,21 +150,24 @@ upload_package_to_db <- function(name, version, title, description,
 insert_riskmetric_to_db <- function(pkg_name, 
     db_name = golem::get_golem_options('assessment_db_name')){
 
-  if (!isTRUE(getOption("shiny.testmode")))
+  if (!isTRUE(getOption("shiny.testmode"))) {
     riskmetric_assess <-
       riskmetric::pkg_ref(pkg_name,
                           source = "pkg_cran_remote",
                           repos = c("https://cran.rstudio.com")) %>%
       dplyr::as_tibble() %>%
       riskmetric::pkg_assess()
-  else
+  } else {
     riskmetric_assess <-
       test_pkg_assess[[pkg_name]]
+  }
+  metric_weights_df <- eventReactive(pkg_name, {
+    dbSelect("SELECT id, name, weight, is_perc FROM metric", db_name)
+  }) 
```

Comment:
What would you like it to be?

Reply:
I would say it's broken because it's introducing reactivity outside of the modules (i.e. inside of a function). I was just curious why the change since you are using `isolate()` to access the elements. I can't see where you are even using the reactivity. Removing it does not seem to have any ill effects. 
[patch.txt](https://github.com/pharmaR/riskassessment/files/11906101/patch.txt)

Reply:
resolved with use get_metric_weights() function